tests/coverage.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Could we put those lines directly in the GitHub Actions file? Or is it useful to execute locally?

Reply:
Maybe we want to execute locally too! If you're writing tests 🤡